docs/content/components/markdown.md

```diff
@@ -6,13 +6,13 @@ source: https://github.com/primer/view_components/tree/main/app/components/prime
 
 <!-- Warning: AUTO-GENERATED file, do not edit. Add code comments to your Ruby instead <3 -->
 
-Use MarkdownComponent to weap markdown content
+Use MarkdownComponent to wrap markdown content
```

Comment:
Is this change supposed to be pulled in on this branch?

Reply:
I think I fixed the comment in another PR but forgot to run `docs:build` :/